README.md

```diff
@@ -10,22 +10,22 @@ result of the OEV bot, there are branches for the previous steps, so one can eas
 On branch [mev](https://github.com/api3dao/oev-v1-compound-bot/tree/mev)
 
 First step is to have an MEV bot that can perform liquidations when the opportunity to do so arises. For more
-information, refer to [OEV Searching](https://docs.api3.org/oev/searchers/).
+information, refer to [MEV Searching](https://docs.api3.org/oev-searchers/in-depth/#from-mev-searching).
```

Comment:
I think OEV Searching is a bit more accurate.

Reply:
Used "From MEV Searching" and also added a link to the oev-v1-compound repo.